pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -188,6 +190,13 @@ def load_data(self):
             )
         )
 
+    def load_uploads(self):
+        """
+        Copies upload fixtures into MEDIA_ROOT
+        """
+        print 'Copying fixtures from uploads directory to MEDIA_ROOT'
+        sh('cp -rf common/test/uploads/* test_root/uploads/')
```

Comment:
Ok I see what you mean. Did you happen to look at whether or not `MEDIA_ROOT` can be set to something else, like a subdir? I'm not too familiar with this setting but wanted to ask.

To your point about avoiding this altogether, I think if we had the right directory to begin with, then we could avoid any copying. What other options are there?

If we must copy over from another directory, I'd rather the copy be more explicit, and I think it should be done via python. But, I want to hear about our other options first because I feel like copying the data over can be brittle.